java/code/src/com/redhat/rhn/frontend/dto/PackageListItem.java

```diff
@@ -110,6 +111,28 @@ public String getPending() {
         return this.pending;
     }
 
+    /**
+     * Gets the retracted.
+     *
+     * @return retracted
+     */
+    public boolean isRetracted() {
+        return retracted;
+    }
+
+    /**
+     * Sets the retracted.
+     *
+     * @param retractedIn the retracted
+     */
+    public void setRetracted(boolean retractedIn) {
+        retracted = retractedIn;
+    }
+
+    public void setRetracted(Integer retractedIn) {
+        retracted = retractedIn == 1;
+    }
```

Comment:
We have code where this is not required. 1/0 are automatically treated as true/false

Reply:
Ok, then disregard my comment above.